modules/tinymce/src/themes/silver/test/ts/phantom/components/customeditor/CustomEditorSchemaTest.ts

```diff
@@ -85,9 +85,7 @@ describe('phantom.tinymce.themes.silver.components.customeditor.CustomEditorSche
         i: /^(?:fizz|buzz)/,
         j: new Blob([ '<a id="a"><b id="b">hey!</b></a>' ], { type: 'text/html' }),
         k: [ 'text', Infinity, true, false ],
-        l: { prop: 'value' },
-        m: new Map([[ 'key1', 'value1' ], [ 'key2', 'value2' ]]),
-        n: new Set([ 1, 2, 3, 4, 5 ])
```

Comment:
NTS: Why did this change

Reply:
I removed Map and Set from the `postMessageable` function, since they're not IE compatible, and this test runs through that function, so.